### ISSUE TITLE
Only record banners in the statistics if they were actually seen in the viewport

### ISSUE THREAD

**Issue by mpitz** [BugBuster1701/contao-banner-bundle#90]
Hi,

is it possible to correctly record the banners in the statistics (i.e. after actually really seeing them in the browsers viewport) if you put them in a slideshow or banner rotation?

It seems like even loading the banner is currently enough for a "view". Even though it wasn't in the browser's viewport.

Greetings

**Comment by BugBuster1701** [comment]
No, this is not possible because the counting is executed on the server side at the moment the banner file is requested.
